packages/chakra-ui/src/IconButton/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default } from "./IconButton";
+export { default } from "./ChakraIconButton";
```

Comment:
Does it make sense to `export * from "./ChakraIconButton";` too?

Reply:
The `default` export is the only thing exported from the module, so it makes no difference. Will change it to 

```
export { default as ChakraIconButton } from "./ChakraIconButton";
```